apps/deploy_monitoring/docker-compose.tezedge_and_explorer.yml

```diff
@@ -14,22 +14,24 @@ services:
         syslog-format: "rfc5424micro"
     volumes:
       - "${TEZEDGE_VOLUME_PATH}:/tmp/tezedge"
+    environment:
+      - TEZOS_STORAGE=index-log-size=5_000_000
     ports:
       - "4927:4927"       # node WS port (required only for tezedge)
-      - "19732:19732"       # node P2P port
+      - "9732:9732"       # node P2P port
       - "18732:18732"     # node RPC port
       - "3030:3030"       # sandbox launcher port
 
   explorer:
-    image: simplestakingcom/tezedge-explorer:latest-release
+    image: tezedge/tezedge-explorer:v1.6.7
     environment:
       - SANDBOX=http://${HOSTNAME}:3030
       - DEBUGGER=http://${HOSTNAME}:17732
       # need a better way to provide such information
-      - API=[{"id":"rust","name":"rust.${HOSTNAME}","http":"http://${HOSTNAME}:18732","ws":"ws://${HOSTNAME}:4927","monitoring":"http://${HOSTNAME}:38732/resources/tezedge","p2p_port":19732,"features":["MONITORING","RESOURCES","MEMPOOL_ACTION","STORAGE_BLOCK","NETWORK_ACTION","LOGS_ACTION"]},{"id":"ocaml","name":"ocaml.${HOSTNAME}","http":"http://${HOSTNAME}:18733","ws":false,"monitoring":"http://${HOSTNAME}:38732/resources/ocaml","p2p_port":9733,"features":["MONITORING","RESOURCES","MEMPOOL_ACTION","NETWORK_ACTION","LOGS_ACTION"]}]
+      - API=[{"id":"rust","type":"tezedge","name":"rust.${HOSTNAME}","http":"http://${HOSTNAME}:18732","p2p_port":9732,"features":[{"name":"ws","url":"ws://${HOSTNAME}:4927"},{"name":"monitoring"},{"name":"resources/system","monitoringUrl":"http://${HOSTNAME}:38732/resources/tezedge"},{"name":"resources/memory","memoryProfilerUrl":"http://${HOSTNAME}:17832"},{"name":"mempool"},{"name":"network"}]},{"id":"ocaml","type":"octez","name":"ocaml.${HOSTNAME}","http":"http://${HOSTNAME}:18733","p2p_port":9733,"features":[{"name":"monitoring"},{"name":"resources/system","monitoringUrl":"http://${HOSTNAME}:38732/resources/ocaml"},{"name":"resources/memory","memoryProfilerUrl":"http://${HOSTNAME}:17832"},{"name":"mempool"},{"name":"network"}]}]
```

Comment:
@tizoc 
could you please remove the part for octez?
```
{"id":"ocaml","type":"octez"...
```

Reply:
done. @bkontur @adonagy so, what do you think of using the multiline formatting here?

Reply:
@tizoc 
very cool, lets do it everywhere this way, it much much better and well-arranged,
and it will safe time with merging and conflicts

Reply:
@tizoc That's perfect :)